### DISCUSSION TITLE
Revisit legal/authorsFile key

### DISCUSSION THREAD

**Discussion by libremente** [publiccodeyml/publiccode.yml#175] [General]
Hi,

I would like to discuss the role of the `legal/authorsFile` key. 
As of today, this is its description:

> Some open-source software adopt a convention of identify the copyright holders through a file that lists all the entities that own the copyright. This is common in projects strongly backed by a community where there are many external contributors and no clear single/main copyright owner. In such cases, this key can be used to refer to the authors file, using a path relative to the root of the repository.

However, the `AUTHORS` file seems to be slowly disappearing from repos nowadays or, when it's present, it's rarely updated with constance. Probably this is due to the role that the various platform (e.g., GitHub) are playing providing the history of the commits. 

Moreover, in the vast majority of the cases the `publiccode.yml` is filled in this way:

` authorsFile: AUTHORS`

which seems to be a bit redundant and IMHO it does not provide much information. 
As such, I would like to know what you think about this key and if it could be revisited/deleted in your view.
Thanks!

**Comment by bfabio** [comment]
I agree, since `AUTHORS` has a well known place if it exists, this key should go. Other variants `AUTHORS{.md,.txt,...}` are one `find` away in the repo anyway.

Care to provide a PR for this?

**Comment by bfabio** [comment]
Proposed in https://github.com/publiccodeyml/publiccode.yml/pull/185
